safehttp/incoming_request_test.go

```diff
@@ -16,11 +16,14 @@ package safehttp_test
 
 import (
 	"context"
-	"github.com/google/go-cmp/cmp"
-	"github.com/google/go-safeweb/safehttp"
 	"net/http"
 	"net/http/httptest"
+	"strings"
 	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-safeweb/safehttp"
+	"github.com/google/go-safeweb/safehttp/safehttptest"
 )
 
 func TestIncomingRequestCookie(t *testing.T) {
```

Comment:
We have some tests names in which we include the file name `TestIncomingRequest...` and some, especially in plugins, where we don't. We should also probably make this uniform.

Reply:
I think the way we are doing it now works well. There will be many tests in `safehttp`, therefore the test names needs to be prefixed with what exactly they are testing. But in the plugins there aren't as many tests that might overlap. So there we can have shorter test names, which is easier to read.